site/content/en/docs/tasks/run/argo_workload.md

```diff
@@ -0,0 +1,167 @@
+
+When creating [Workflow](https://argo-workflows.readthedocs.io/en/latest/workflow-concepts/),
+[WorkflowTemplate](https://argo-workflows.readthedocs.io/en/latest/workflow-templates/),
+[ClusterWorkflowTemplate](https://argo-workflows.readthedocs.io/en/latest/cluster-workflow-templates/),
+or [CronWorkflow](https://argo-workflows.readthedocs.io/en/latest/cron-workflows/) resources for
+Kueue, take into consideration the following aspects:
+
+### a. Targeting a single LocalQueue
+
+If you want the entire workflow to target a single [local queue](/docs/concepts/local_queue),
+it should be specified in the `spec.podMetadata` section of the Workflow configuration.
+
+```yaml
+apiVersion: argoproj.io/v1alpha1
+kind: Workflow
+metadata:
+  generateName: hello-world-
+spec:
+  entrypoint: whalesay
+  podMetadata:
+    labels:
+      kueue.x-k8s.io/queue-name: user-queue # All pods will target user-queue
+  templates:
+    - name: whalesay
+      container:
+        image: docker/whalesay
+        command: [ cowsay ]
+        args: [ "hello world" ]
+        resources:
+          limits:
+            memory: 32Mi
+            cpu: 100m
+```
+
+### b. Targeting a different LocalQueue per template
+
+If prefer to target a different [local queue](/docs/concepts/local_queue) for each step of your Workflow,
+you can define the queue in the `spec.templates[].metadata` section of the Workflow configuration.
+
+In this example `hello1` and `hello2a` will target `user-queue` and `hello2b` will
+target `user-queue-2`.
+
+```yaml
+apiVersion: argoproj.io/v1alpha1
+kind: Workflow
+metadata:
+  generateName: steps-
+spec:
+  entrypoint: hello-hello-hello
+
+  templates:
+  - name: hello-hello-hello
+    steps:
+    - - name: hello1            # hello1 is run before the following steps
+        template: whalesay
+        arguments:
+          parameters:
+          - name: message
+            value: "hello1"
+    - - name: hello2a           # double dash => run after previous step
+        template: whalesay
+        arguments:
+          parameters:
+          - name: message
+            value: "hello2a"
+      - name: hello2b           # single dash => run in parallel with previous step
+        template: whalesay-queue-2
+        arguments:
+          parameters:
+          - name: message
+            value: "hello2b"
+
+  - name: whalesay
+    metadata:
+      labels:
+        kueue.x-k8s.io/queue-name: user-queue # Pods from this template will target user-queue
+    inputs:
+      parameters:
+      - name: message
+    container:
+      image: docker/whalesay
+      command: [cowsay]
+      args: ["{{inputs.parameters.message}}"]
+
+  - name: whalesay-queue-2
+    metadata:
+      labels:
+        kueue.x-k8s.io/queue-name: user-queue-2 # Pods from this template will target user-queue-2
+    inputs:
+      parameters:
+      - name: message
+    container:
+      image: docker/whalesay
+      command: [cowsay]
+      args: ["{{inputs.parameters.message}}"]
```

Comment:
does it need to be expanded to make the example runnable?

Reply:
I think this is runnable.